index.html

```diff
@@ -29,62 +29,58 @@
               </a>
             </li>
             <li>
-              <a href="https://www.facebook.com/">
-                English
-              </a>
+              <a href="https://www.facebook.com/"> English </a>
             </li>
             <li>
-              <a href="https://www.facebook.com/">
-               My Page
-              </a>
+              <a href="https://www.facebook.com/"> My Page </a>
             </li>
             <li>
-              <a href="https://www.facebook.com/">
-                Logout
-              </a>
+              <a href="https://www.facebook.com/"> Logout </a>
             </li>
           </ul>
         </nav>
         <nav class="navbar">
-
-         <div class="mobile">
-             <span class="menuIcon">
-                <i class="fas fa-bars icon"></i>
-              </span>
-              <span class="close">
-                <i class="fas fa-times"></i>
-              </span>
-         </div>
+          <div class="mobile">
+            <span class="menuIcon">
+              <i class="fas fa-bars icon"></i>
+            </span>
+            <span class="close">
+              <i class="fas fa-times"></i>
+            </span>
+          </div>
           <span class="logo-header">
             <a href="#">
-              <img src="https://res.cloudinary.com/ujuzi/image/upload/v1643800152/MICROVERSE/capstone-project-html-css-javascript/conference_logo_yeqedt.png" alt="logo" />
+              <img
+                src="https://res.cloudinary.com/ujuzi/image/upload/v1643800152/MICROVERSE/capstone-project-html-css-javascript/conference_logo_yeqedt.png"
+                alt="logo"
+              />
             </a>
           </span>
-            <div id="navbar-container" class="navbar-container">
-                <div class="desk-navbar-menu">
-                  <ul class="desk-navbar-menu-list">
-                    <li class="desk-navbar-menu-item">
-                      <a href="./index.html">Home</a>
-                    </li>
-                    <li class="desk-navbar-menu-item">
-                      <a href="./src/css/about.html">About</a>
-                    </li>
-                    <li class="desk-navbar-menu-item">
-                      <a href="#">Projects</a>
-                    </li>
-                    <li class="desk-navbar-menu-item">
-                      <a href="#">Contact</a>
-                    </li>
-                  </ul>
-                </div>
-              </div>
-              
+          <div id="navbar-container" class="navbar-container">
+            <div class="desk-navbar-menu">

```

Comment:
- [ ] Please notice that once mobile `navbar` is open, it does not disappear when the screen size gets bigger: 

https://user-images.githubusercontent.com/77038610/152655363-833adf13-9cf0-4b94-be69-10f1419cefdd.mp4

I suggest you to consider fixing this behavior. 

- [ ] Users can scroll the page when the `navbar` is open, this is not a good practice, please consider adding `overflow: hidden` to body while mobile `nav` is open. 
- [ ] Lastly, `navbar` is transparent and users can see the page, I recommend you making the background dark or white, so that users could not see other sections while  mobile `navbar` is open.

Reply:
Sure I working On this

Reply:
Here is the new screen screenshot on menu open
https://res.cloudinary.com/ujuzi/image/upload/v1644093721/Cars%20Images%20project/hack2_serto6.png